### ISSUE TITLE
Validate prop not accessible in field custom component

### ISSUE THREAD

**Issue by VarshaJadhav** [erikras/redux-form#4090]
I'm using **redux-form: "7.1.2"**

`import CustomComponent from 'custom-component'
import { required } from 'validation'; 

<Field 
component={CustomComponent}
name="firstname"
validate=[required]
/>
`
When I'm trying to access props in CustomComponent by **this.props** but validate prop is not available in CustomComponent.

**What I want:**
Which validation functions are applied to firstname field

**Comment by VarshaJadhav** [comment]
Yes we can pass custom props to field and that will be available in CustomComponent.

But in my case, I've used Field multiple times at different places so instead of passing custom prop to each field Is there any way to access validate prop in CustomComponent. (because validate is already defined in each component)

**Comment by yoginair** [comment]
@gustavohenke : I also got the same problem...Any solution ?

**Comment by DonovanCharpin** [comment]
I agree, having to do something like the example bellow is not optimal.
```
<Field 
  component={CustomComponent}
  name="firstname"
  validate={[required()]}
  required
/>
```
It would be nice if the validate props would be given to the custom field. In my case, I cannot add an asterisk for required Field. I could create an HOC for the Field component but I think it should be given to the custom field to manage different rendering regarding the validation. The required props could be given as meta if there is a required in the validate props but it doesn't cover others use cases.
